backend/app/crud/prompts.py

```
@@ -0,0 +1,191 @@
+from uuid import UUID
+
+from fastapi import HTTPException
+from sqlmodel import Session, and_, func, select
+
+from app.core.util import now
+from app.models import (
+    Prompt,
+    PromptCreate,
+    PromptUpdate,
+    PromptVersion,
+    PromptWithVersion,
```

Ruff (0.12.2)

`app.models.PromptWithVersion` imported but unused

Remove unused import

(F401)